samples/logging_s3/pipeline_log_to_s3_by_fluentd_recommend/fluent-config.yaml

```
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluent-config
```

Typo for the configmap name
> 
```suggestion
  name: fluentd-config
```

---

oh, it's a typo, Thanks @tomcli, the fluentd-config configmap was created in my ENV before, that's why I didn't test it out.

But I have to use fluent-config to align with ${FLUENTD_CONF}
# env |grep FLUENTD_CONF
FLUENTD_CONF=fluent.conf